### DISCUSSION TITLE
"Minimal" theme incompatibility

### DISCUSSION THREAD

**Discussion by AndrewGreenFrost** [vslinko/obsidian-outliner#205] [Archive]
With applied "Minimal" theme appears wrong lines in outlining. With default theme everything is OK. 
Please any advice how to fix this incompatibility with "Minimal" theme.
![Minimal](https://user-images.githubusercontent.com/90498977/136648733-cd107911-7046-4c27-8df8-5405beb38250.jpg)

**Comment by vslinko** [comment]
Unfortunately, only the default theme is supported.
